packages/grant-explorer/src/features/contributors/ViewContributionHistory.tsx

```diff
@@ -1,29 +1,21 @@
 import { useAccount } from "wagmi";
 import { DetailedVote as Contribution } from "allo-indexer-client";
-import {
-  ChevronRightIcon,
-  InformationCircleIcon,
-} from "@heroicons/react/24/solid";
 import { lazy, useMemo } from "react";
-import { Link, useParams } from "react-router-dom";
-import { ethers } from "ethers";
+import { useParams } from "react-router-dom";
 import { PayoutToken } from "../api/types";
-import { CHAINS, getChainIds, getTxExplorerTxLink } from "../api/utils";
+import { getChainIds, payoutTokens } from "../api/utils";
 import Navbar from "../common/Navbar";
-const DonationHistoryBanner = lazy(
-  () => import("../../assets/DonationHistoryBanner")
-);
 import blockies from "ethereum-blockies";
 import CopyToClipboardButton from "../common/CopyToClipboardButton";
 import Footer from "common/src/components/Footer";
-import { payoutTokens } from "../api/utils";
-import { ArrowTopRightOnSquareIcon } from "@heroicons/react/24/outline";
-import { Button } from "common/src/styles";
-import ReactTooltip from "react-tooltip";
 import Breadcrumb, { BreadcrumbItem } from "../common/Breadcrumb";
 import { useContributionHistory } from "../api/round";
 import { StatCard } from "../common/StatCard";
-import { ChainId } from "common";
+import { DonationsTable } from "./DonationsTable";
+
+const DonationHistoryBanner = lazy(
+  () => import("../../assets/DonationHistoryBanner")
```

Comment:
so that we don't include the banner in the main bundle. if you go to a specific round page, you don't want to be downloading a huge image that you won't even see.

Reply:
In these cases I'd suggest then adding a comment so an unfamiliar reader isn't tempted to "fix" it.